ci-operator/jobs/devfile/integration-tests/devfile-integration-tests-main-periodics.yaml

```diff
@@ -12,10 +12,10 @@ periodics:
   labels:
     ci-operator.openshift.io/cloud: aws
     ci-operator.openshift.io/cloud-cluster-profile: aws-devfile
-    ci-operator.openshift.io/variant: v4.10.console
+    ci-operator.openshift.io/variant: v4.11.console
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-devfile-integration-tests-main-v4.10.console-e2e-aws-console-periodic
+  name: periodic-ci-devfile-integration-tests-main-v4.11.console-e2e-aws-console-periodic
```

Comment:
v4.11? (and same for odo down below)

Reply:
https://github.com/openshift/release/pull/53140#discussion_r1638720416